public/pages/TopNQueries/TopNQueries.tsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import { useHistory, useLocation, Switch, Route, Redirect } from 'react-router-dom';
+import { EuiTab, EuiTabs, EuiTitle } from '@elastic/eui';
+import { FormattedMessage } from '@osd/i18n/react';
+import QueryInsights from '../QueryInsights/QueryInsights';
+
+const TopNQueries = () => {
+  const history = useHistory();
+  const location = useLocation();
+
+  const tabs: Array<{ id: string; name: string; route: string }> = [
+    {
+      id: 'topNQueries',
+      name: 'Top N queries',
+      route: 'queryInsights',
+    },
+    {
+      id: 'configuration',
+      name: 'Configuration',
+      route: 'configuration',
+    },
+  ];
+
+  const onSelectedTabChanged = (route: string) => {
+    const { pathname: currPathname } = location;
+    if (!currPathname.includes(route)) {
+      history.push(route);
+    }
+  };
+
+  const renderTab = (tab: { route: string; id: string; name: string }) => (
+    <EuiTab
+      onClick={() => onSelectedTabChanged(tab.route)}
+      isSelected={location.pathname.includes(tab.route)}
+      key={tab.id}
+    >
+      {tab.name}
+    </EuiTab>
+  );
+
+  return (
+    <div style={{ padding: '35px 35px' }}>
+      <Switch>
+        <Route
+          exact
+          path="/queryInsights"
```

Comment:
Are those paths strings supposed to be the same as what's defined in:
```
const tabs: Array<{ id: string; name: string; route: string }> = [
    {
      id: 'topNQueries',
      name: 'Top N queries',
      route: 'queryInsights',
    },
    {
      id: 'configuration',
      name: 'Configuration',
      route: 'configuration',
    },
  ];
```
If so we should create constants for them.